src/routes/templates/templates.json

```diff
@@ -1,4 +1,16 @@
 [
+	{
+    		"title": "svelte-app",
```

Comment:
`svelte-app` is a bit too generic for a template name.

I believe that (almost) all templates are for building an app, so they all are a "Svelte App".

Can you choose a name that better describe your template ?